<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>221</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="optionsLayout"/>
   </widget>
   <widget class="QFrame" name="imagesFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>611</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="imagesPanel" name="imagePanel" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>611</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="buttonsFrame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>191</width>
      <height>611</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>191</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Transformações OpenCV</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="grayScaleButton_2">
          <property name="text">
           <string>Apply Gray Scale</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="hsvFilterButton_2">
          <property name="text">
           <string>HSV Color Filter</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="bilateralFilterButton_2">
          <property name="text">
           <string>Bilateral Filtering</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="gaussianBlurButton_2">
          <property name="text">
           <string>Gaussian Blur</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="medianBlurButton_2">
          <property name="text">
           <string>Median Blur</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="simplethresholdingButton_2">
          <property name="text">
           <string>Simple Thresholding</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="adaptiveThresholdingButton_2">
          <property name="text">
           <string>Adaptive Thresholding</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="subtractImageButton_2">
          <property name="text">
           <string>Subtract from Original Gray Scale</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="morphologyOperatorButton_2">
          <property name="text">
           <string>Morphology Operator</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="openCVSobelButton_2">
          <property name="text">
           <string>OpenCV Sobel Derivatives</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="customSobelButton_2">
          <property name="text">
           <string>Custom Sobel + Threshold</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="loadImageButton_2">
          <property name="text">
           <string>Load Image</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="undoButton">
          <property name="text">
           <string>Undo Operation</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="findContoursButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Contours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawContoursButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Contours</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>261</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>                   Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="logTextBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBoxRecordSteps">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Steps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSaveStrategy">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Strategy</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_CV_image_transformations">
    <property name="title">
     <string>Open CV image transformations</string>
    </property>
   </widget>
   <addaction name="menuOpen_CV_image_transformations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imagesPanel</class>
   <extends>QWidget</extends>
   <header>imagespanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
